101-backup-la-reporting/nestedtemplates/azure_alerts.json

```diff
@@ -0,0 +1,547 @@
+{
+   "$schema":"https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+   "contentVersion":"1.0.0.0",
+   "parameters":{
+      "workspaceLocation":{
```

Comment:
update this to location in the nested templates as well

Reply:
Updated